src/edit_view.rs

```diff
@@ -364,23 +427,31 @@ impl EditView {
                 match name.as_ref() {
                     "font_size" => {
                         if let Some(font_size) = value.as_u64() {
-                            let pango_ctx = self.da.get_pango_context().unwrap_or_else(|| {
-                                panic!("{}", &gettext("Failed to get Pango context"))
-                            });
+                            let pango_ctx = self
+                                .view_item
+                                .edit_area
+                                .get_pango_context()
+                                .unwrap_or_else(|| {
+                                    panic!("{}", &gettext("Failed to get Pango context"))
+                                });
```

Comment:
Could we use the new get_pango_ctx func here?

Reply:
Good eye!